lib/hydra/works/services/determine_mime_type.rb

```diff
@@ -0,0 +1,24 @@
+module Hydra::Works
+  class DetermineMimeType
+    # Determines the mime type for a given file
+    # @param [IO, File, Rack::Multipart::UploadedFile, #read] file
+    # @param [String, NilClass] original_name of the file
+    # @return [String]
+    class << self
```

Comment:
I don't feel strongly about this, especially given we use different patterns in the hydra-works codebase, but I have a slight preference for the service object design that uses a class method to create an instance. Like so:

https://github.com/projecthydra/hydra-works/blob/master/lib/hydra/works/services/characterization_service.rb#L9-L21

That way you can make use of `attr_accessor` instead of passing around the same arguments (e.g., `file` and `original_name`). Like I said, slight preference.